### PR TITLE
CLI is complete, Tests are working.

### PR DESCRIPTION
**Pull request by swsnider**
GUI is a stub -- pay no attention to it :)

### CODE REVIEW

File: EVE-Val SQL to YAML Tool/SQL To YAML CLI/Options.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+using CommandLine.Text;
+
+namespace SQL_To_YAML_CLI
+{
+    class Options
+    {
+        [Option(null, "connection_string", DefaultValue="Data Source=localhost; Initial Catalog=ebs_DATADUMP; Integrated Security=SSPI;", HelpText = "The database connection string to use.")]
+        public string connectionString { get; set; }
+
+        [Option("t", "table", DefaultValue="*", HelpText="The database table to extract. Defaults to '*'.")]
```
Comment (ayust) [inline EVE-Val SQL to YAML Tool/SQL To YAML CLI/Options.cs]: The default being a wildcard implies that (a) wildcards can be used and (b) multiple tables can be extracted at the same time. If this is the case, `table(s)` would probably be more appropriate, and a mention of what wildcards are allowed.

File: EVE-Val SQL to YAML Tool/SQL To YAML CLI/Options.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+using System.Linq;
+using System.Text;
+using CommandLine;
+using CommandLine.Text;
+
+namespace SQL_To_YAML_CLI
+{
+    class Options
+    {
+        [Option(null, "connection_string", DefaultValue="Data Source=localhost; Initial Catalog=ebs_DATADUMP; Integrated Security=SSPI;", HelpText = "The database connection string to use.")]
```
Comment (ayust) [inline EVE-Val SQL to YAML Tool/SQL To YAML CLI/Options.cs]: This is really long. Does C# allow this to be linewrapped?

(In general, it's best to keep code width below the ~120 characters visible in GitHub diff views.)

**Comment by swsnider** [comment]
PTAL.

**Comment by ayust** [comment]
LGTM

**Pull request merged**